CONTRIBUTING-students.md

```diff
@@ -6,44 +31,37 @@ The 2016 student application window is March 14th to 25th.
 [The Python Software Foundation Wiki](https://wiki.python.org/moin/SummerOfCode/2016#Students)
 lists a few things you can do to get started in free and open source software.
 
-## Getting Started
-
-**Please use the instructions of the project/organization
-you are interested in work with.**
-
-## biocore
-
-Read [default instructions](#default-instructions).
-
-## DyND
-
-Read [default instructions](#default-instructions).
-
-## Gensim
-
-Read [default instructions](#default-instructions).
-
-## JuliaOpt
+## Am I experienced enough?
 
-Read [default instructions](#default-instructions).
+The answer is generally: **Yes**. 
+We value creativity, intelligence and enthusiasm above specific knowledge of the libraries or algorithms we use. 
+We think that an interested and motivated student who is willing to learn is more valuable than anything else.
+The range of available projects should suit people with different backgrounds, both with and without prior experience of computational chemistry.
+At the same time if you have experience using your project of choice or one of it's dependencies (e.g. language) make sure to let us know about that as well.
 
-## matplotlib
+## Our Expectations From Students
 
-Read [default instructions](#default-instructions).
+**Communication**
 
-## Pandas
+- Write a short report for us every second week, this can be a blog post
+- Commit early and commit often! Push to a public repository (e.g. github) so that we can see and review your work.
+- Actively work on our project timeline and communicate with us during the community bonding period
+- Communicate every working day with your mentor. Just say "Hello" if you like. It can be via email, skype, github comments, etc
```

Comment:
I changed that the communication should be preferable in public on a project channel. So that the development happens like any other feature development.